integration-tests/oauth2/src/main/java/com/okta/spring/tests/oauth2/implicit/BasicImplicitFlowApplication.java

```diff
@@ -41,19 +55,60 @@ protected MethodSecurityExpressionHandler createExpressionHandler() {
         }
     }
 
-    @GetMapping("/")
-    @PreAuthorize("#oauth2.hasScope('email')")
-    public String getMessageOfTheDay(Principal principal) {
-        return "The message of the day is boring: " + principal.getName();
+    @Configuration
+    @Order(0)
+    static class ResourceSecurityConfigurer extends ResourceServerConfigurerAdapter {
+
+        @Override
+        public void configure(HttpSecurity http) throws Exception {
+            http.authorizeRequests()
+                    .antMatchers(HttpMethod.OPTIONS,"/**").permitAll()
```

Comment:
I see that this is in tests, but why are we disabling this on all paths?

Reply:
`OPTIONS` is permitted on all paths without first requiring auth.